sycl/doc/design/OffloadDesign.md

```diff
@@ -236,22 +245,22 @@ are given further below.
 | Option Name                  | Purpose                                      |
 |------------------------------|----------------------------------------------|
 | `--fpga-link-type=<arg>`     | Tells the link step to perform 'early' or 'image' processing to create archives for FPGA |
-| `--parallel-link-sycl=<arg>` | Provide the number of parallel jobs that will be used when processing with `llvm-foreach` |
+| `--parallel-link-sycl=<arg>` | Provide the number of parallel jobs that will be used when processing split jobs |
 | `--no-sycl-device-lib=<arg>` | Provide the list of device libraries to restrict from linking during device link |
```

Comment:
Do we want to repeat this option here?

Reply:
This corresponds to the 'list' of device libraries which are now controlled by the driver.  I will remove this.